### DISCUSSION TITLE
Does RESUME_CHECKPOINT support loading specific safetensors file for SD3 full finetuning?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [bghira/SimpleTuner#498] [Q&A]
Does RESUME_CHECKPOINT support loading specific safetensors file for SD3 full finetuning, and not the whole transformers folder? Can I give it any path? Does it require a specific folder structure and files besides safetensors file?
I save only safetensors files from training, so I'm not sure how to continue the training from previous safetensors on a new and clean server.

**Comment by bghira** [comment]
it has to have everything: scheduler, transformer, model_index.json

it should have the first two text encoders and T5 can be pointed anywhere via `--pretrained_t5_model_name_or_path` but that needs the text_encoder / tokenizer subfolders

you wouldn't use resume_checkpoint for anything other than loading a simpletuner checkpoint, because it contains state information / state bins that normal ckpts don't, such as the optimizer / random state pickles.

if you wanted to load a single transformers file you'd have to create a diffusers type repo for it on the hub first, ideally.
